scopes/lanes/lanes/lane.cmd.ts

```diff
@@ -295,7 +293,7 @@ export class LaneRemoveCmd implements Command {
   group = 'collaborate';
   alias = '';
   options = [
-    ['r', 'remote', 'remove a remote lane (in the lane arg, use remote/lane-id syntax)'],
+    ['r', 'remote', 'remove a remote lane (in the lane arg, use remote/lane-id syntax)'], // @david please add an example of the syntax here
```

Comment:
e.g. `bit lane remove owner.org/my-lane --remote`

Reply:
cheers

by the way, does this command just remove a lane from the workspace, or actually delete it from the remote?

Reply:
and if the latter, does it require an export for the deletion to take effect?

Reply:
It doesn't remove it from the workspace, only from the remote. and it doesn't require an export. (it's different than components..)

Reply:
ok, so i'm changing the description to say `delete` rather than remove

Reply:
another clarification - if you run this **without** the --remote flag, what does it do?
remove from the local workspace? from the remote? both?

Reply:
without the `--remote`, it removes it from the local scope only. (you can run it when you are checked out to another lane or to main).

Reply:
ok, so without the remote flag it just removes from the local scope (no effect on the remote scope).
with the remote flag it deletes remotely

I suggest we implement the same remove/delete commands for lanes then like we have for components, i.e. that deletion happens from `bit lane delete` and not `bit lane remove` as it's much clearer that way.
Not in the scope of this PR though